perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/OwnersManagerMethod.java

```diff
@@ -58,13 +58,25 @@ public Owner call(ApiCaller ac, Deserializer parms) throws PerunException {
 	 * @param owner int Owner <code>id</code>
 	 * @return Object Always null
 	 */
+	/*#
+	 * Forcefully deletes an owner.
+	 *
+	 * @param owner int Owner <code>id</code>
+	 * @param force boolean Force must be true
+	 * @return Object Always null
+	 */
 	deleteOwner {
 		@Override
 		public Owner call(ApiCaller ac, Deserializer parms) throws PerunException {
 			parms.stateChangingCheck();
 
-			ac.getOwnersManager().deleteOwner(ac.getSession(),
+			if (parms.contains("force")) {
+				ac.getOwnersManager().deleteOwner(ac.getSession(),
+					ac.getOwnerById(parms.readInt("owner")), parms.readBoolean("force"));
+			} else {
+				ac.getOwnersManager().deleteOwner(ac.getSession(),
 					ac.getOwnerById(parms.readInt("owner")));
+			}
```

Comment:
To avoid the code duplication, you can do something like this:
```suggestion
			boolean force = false;
			if (parms.contains("force")) {
			    force = parms.readBoolean("force");
			}
		    ac.getOwnersManager().deleteOwner(ac.getSession(),
					ac.getOwnerById(parms.readInt("owner")), force);
```